interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,451 @@
+              </constraint>
+              <multi/>
+            </properties>
+          </leafNode>
+          <leafNode name="local-address">
+            <properties>
+              <help>IP address</help>
+              <valueHelp>
+                <format>ipv4</format>
+                <description>IPv4 address</description>
+              </valueHelp>
+              <valueHelp>
+                <format>ipv6</format>
+                <description>IPv6 address</description>
+              </valueHelp>
+              <valueHelp>
+                <format>host</format>
+                <description>Host</description>
+              </valueHelp>
+              <constraint>
+                <validator name="ip-host"/>
+              </constraint>
+              <multi/>
+            </properties>
+          </leafNode>
+          <leafNode name="port">
+            <properties>
+              <help>Port Option</help>
+              <constraint>
+                <validator name="numeric" argument="--range 1-65535"/>
+              </constraint>
+            </properties>
+            <defaultValue>655</defaultValue>
+          </leafNode>
+          <leafNode name="bind-interface">
+            <properties>
+              <help>Bind To Interface</help>
+              <completionHelp>
+                <script>${vyos_completion_dir}/list_interfaces.py --broadcast</script>
+              </completionHelp>
+            </properties>
+          </leafNode>
+          <tagNode name="connect">
+            <properties>
+              <help>Connect To Peer Node Name</help>
+            </properties>
+            <children>
+              <leafNode name="conf-path">
+                <properties>
+                  <help>The host public key configuration file to be connected</help>
+                  <valueHelp>
+                    <format>user@ip:path</format>
+                    <description>SCP Protocol Copy</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>path</format>
+                    <description>Local Protocol Copy</description>
+                  </valueHelp>
+                </properties>
+              </leafNode>
+            </children>
+          </tagNode>
+          #include <include/address-ipv4-ipv6-dhcp.xml.i>
+          #include <include/interface-description.xml.i>
+          #include <include/dhcp-options.xml.i>
+          #include <include/dhcpv6-options.xml.i>
+          #include <include/interface-disable-link-detect.xml.i>
+          #include <include/interface-disable.xml.i>
+          #include <include/interface-vrf.xml.i>
+          <node name="encryption">
+            <properties>
+              <help>Data Encryption settings</help>
+            </properties>
+            <children>
+              <leafNode name="cipher">
+                <properties>
+                  <help>Standard Data Encryption Algorithm(default:aes-256-cbc)</help>
```

Comment:
There should be spaces for readaibility. "Encryption algorithm (default: aes-256-cbc)"

Reply:
done